CoursesManager.UI/Factory/ViewModelFactory.cs

```diff
@@ -23,6 +24,18 @@ public class ViewModelFactory
         private readonly IAddressRepository _addressRepository;
         private readonly IMessageBroker _messageBroker;
         private readonly IDialogService _dialogService;
+        private readonly IConfigurationService _configurationService;
+        private string _dbName;
+        private string _dbServer;
+        private string _dbPort;
+        private string _dbUser;
+        private string _dbPassword;
+        private string _mailServer;
+        private string _mailPort;
+        private string _mailUser;
+        private string _mailPassword;
+        private EnvModel _appConfig;
```

Comment:
Als je dat ding in configurationservice hebt gefixt kan dit hier daarna ook weg

Reply:
Done